app/views/includes/head.jade

```diff
@@ -12,14 +12,14 @@ head
   meta(name="viewport", content="initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=0")
 
   link(href='/img/icons/favicon.ico', rel='shortcut icon', type='image/x-icon')
-  link(href='http://fonts.googleapis.com/css?family=Lobster', rel='stylesheet', type='text/css')
+  link(href='https://fonts.googleapis.com/css?family=Lobster', rel='stylesheet', type='text/css')
 
   meta(property='fb:app_id', content='APP_ID')
   meta(property='og:title', content='#{appName} - #{title}')
   meta(property='og:description', content='Cards for Humanity - A game for horrible people desperately trying to do good')
   meta(property='og:type', content='website')
-  meta(property='og:url', content='http://cfh.io')
-  meta(property='og:image', content='http://cfh.io/img/screenshot-small.png')
+  meta(property='og:url', content='https://cfh.io')
```

Comment:
use http here. I don't think cfh is hosted on https

Reply:
Changes have been effected.